config/docker-compose.yml

```
@@ -104,11 +104,10 @@ services:
 #    Debug port closed for security reasons, be careful if you open it 
 #     - "4100:5005"
     volumes:
-     - "../certificates:/certificates_to_add"
+     - "./certificates:/certificates_to_add"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)